### ISSUE TITLE
Rust does not provide default Debug/Eq/PartialEq traits for arrays having sizes greater then 32

### ISSUE THREAD

**Issue by madebr** [phimuemue/rs_plain_enum#1]
This code fails.
```rust
#[macro_use] extern crate plain_enum;
plain_enum_mod!{number_mod, Numbers {
    N00,    N01,    N02,    N03,    N04,    N05,    N06,    N07,    N08,    N09,
    N10,    N11,    N12,    N13,    N14,    N15,    N16,    N17,    N18,    N19,
    N20,    N21,    N22,    N23,    N24,    N25,    N26,    N27,    N28,    N29,
    N30,    N31,    N32,
}}
```
I believe [this line](https://github.com/phimuemue/rs_plain_enum/blob/5c55888397221386295748de75e94deebafe4c49/src/lib.rs#L158) is the offender.

**Comment by phimuemue** [comment]
I agree that this is unfortunate. However, I am not sure whether we should workaround this within `plain_enum`. I would assume that Rust will address this issue (see [here](https://internals.rust-lang.org/t/impl-traits-for-specific-array-lengths-above-32/5130) or [here](https://github.com/rust-lang/rfcs/issues/1930)), so I am tempted to live with the restriction in `plain_enum` for now, hoping that the problem will disappear.

Do you think there is an easy alternative that we could implement?

**Comment by sugar700** [comment]
I think it may be fixed by now.
